### PR TITLE
check for token expiration

### PR DESCRIPTION
**Pull request by adammeyer**

### CODE REVIEW

File: src/authClient.js [hunk 1]
```diff
@@ -25,7 +26,14 @@ export default (type, params) => {
       case AUTH_CHECK: {
         const sessionToken = sessionStorage.getItem('sessionToken');
         if(sessionToken){
-          return Promise.resolve();
+          const session = jwt_decode(sessionToken);
+          const isExpired = (new Date().getTime() / 1000) - session.exp >= 0;
+
+          if(!isExpired){
+            return Promise.resolve();
```
Comment (Omicron7) [inline src/authClient.js]: Should you reject a promise if it's expired?

Comment (wrandall22) [reply src/authClient.js]: It appears to go back through the auth process if it is expired.

**Comment by wrandall22** [comment]
Did you get a chance to test this?

**Comment by adammeyer** [comment]
Yes, I did locally.

**Pull request merged**